README.md

```diff
@@ -23,17 +23,11 @@ Version 1.2.1
 - Fixed issue opnsense 19.7 log format change for ICMPv6 and TCP/UDP logs.
 ```
 
-## Supported Sourcetypes
-```
-opnsense:filterlog
-opnsense:dhcpd
-opnsense:suricata
-opnsense:squid
-opnsense:cron
-opnsense:unbound
-opnsense:lighttpd
-opnsense:access
-```
+## Universal Forwarder
+
+Use the FreeBSD universal forwarder on OpnSense at [splunk.com](https://www.splunk.com/en_us/download/universal-forwarder.html#tabs/freebsd)
```

Comment:
I would advise against putting a UF on the firewall itself. Typically, syslog is utilized to send the logs to a syslog server which Splunk is monitoring.

Reply:
OK cool, it wasn't clear on what was expected from the readme so went with UF, ignore this if you disagree but suggest linking to docs for syslog configuration for new users

Reply:
That makes sense. I can put a brief walk-through on how to send data to a syslog server. Thanks for the feedback on this.